### PR TITLE
Feature/books component

### PR DESCRIPTION
**Pull request by PalomaRmMz**
### Changes made
Add BookComponent
Add Query GET_BOOKS
Show list of books in BookComponent
Rewrite CardBookComponent for use Props
Resolve conflics with styles

### Where to start
src\books\books-component.js
src\cards-book\card-book-component.js

### How to check the changes?
In the page `<Libros>` and page `<Home>`

### CODE REVIEW

File: src/books/books-component.js [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+      coverUrl
+    }
+  }
+`;
+
+function ExchangeRates() {
+  const { loading, data, error } = useQuery(GET_BOOKS);
+
+  if (loading) {
+    console.log(loading);
```
Comment (NemeSnz) [inline src/books/books-component.js]: This line should not be commited

File: src/home/home-component.js [hunk 2]
```diff
@@ -1,18 +1,63 @@
...
+      coverUrl
+    }
+  }
+`;
+
+function ExchangeRates() {
+  const { loading, data, error } = useQuery(GET_BOOKS);
+
+  if (loading) {
+    console.log(loading);
```
Comment (NemeSnz) [inline src/home/home-component.js]: This line should not be committed

File: src/home/home-component.js [hunk 3]
```diff
@@ -1,18 +1,63 @@
...
+            Lorem, ipsum dolor sit amet consectetur adipisicing elit. Id,
+            architecto quis. Nihil obcaecati quidem omnis sed minus vel
+            reiciendis tempora eum, rem, debitis fugit dolor sit dict a fuga,
+            quisquam adipisci.
           </p>
         </div>
       </header>
-      <CardsBookComponent></CardsBookComponent>
+      <ExchangeRates />
+      <Link to="../books">
```
Comment (NemeSnz) [inline src/home/home-component.js]: Change this path to `./books`

**Review by NemeSnz — CHANGES_REQUESTED** [review]

File: src/home/home-component.js (hunk 3 above)
Comment (PalomaRmMz) [reply src/home/home-component.js]: Resolved

File: src/home/home-component.js (hunk 2 above)
Comment (PalomaRmMz) [reply src/home/home-component.js]: Resolved

File: src/books/books-component.js (hunk 1 above)
Comment (PalomaRmMz) [reply src/books/books-component.js]: Resolved

**Review by NemeSnz — APPROVED** [review]

**Pull request merged**